Remotion/Core/Collections.DataStore.UnitTests/ConcurrentDataStoreTest.cs

```diff
@@ -11,6 +12,7 @@
 namespace Remotion.Collections.DataStore.UnitTests
 {
   [TestFixture]
+  [SuppressMessage("Usage", "RMCORE0001: Use Safecontext instead of typical API")]
```

Comment:
I think we should use `pragma` instead instead of `SupressMessage` in this test class.